### PR TITLE
OSDOCS-7998: update config yaml path and text

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-7998

Link to docs preview:
https://65681--docspreview.netlify.app/microshift/latest/microshift_configuring/microshift-using-config-tools#microshift-yaml-default_microshift-configuring

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: modules/microshift-config-yaml.adoc [hunk 1]
```diff
@@ -6,15 +6,20 @@
 [id="microshift-config-yaml_{context}"]
 = Using a YAML configuration file
 
-{product-title} searches for a configuration file in the user-specific directory, `~/.microshift/config.yaml`, then the system-wide `/etc/microshift/config.yaml` directory. You must create the configuration file and specify any settings that should override the defaults before starting {product-title}.
-
-//include admonition snippet about waiting for greenboot
+{product-title} searches for a configuration file in the system-wide `/etc/microshift/config.yaml` directory. You must create the configuration file and specify any settings that should override the defaults before starting {product-title}.
```
Comment (eggfoobar) [inline modules/microshift-config-yaml.adoc:9]: small change to this,

> the system-wide `/etc/microshift/config.yaml` directory

to this

> the system-wide `/etc/microshift` directory for a file named `config.yaml`

File: modules/microshift-config-yaml.adoc [hunk 2]
```diff
@@ -44,21 +49,22 @@ debugging:
 <8> Subject Alternative Names for API server certificates.
 <9> Log verbosity. Valid values for this field are `Normal`, `Debug`, `Trace`, or `TraceAll`.
 
+[id="microshift-yaml-custom_{context}"]
+== Custom settings
+To create custom configurations, you must create a `config.yaml` file in the `/etc/microshift/config.yaml` directory, then change any settings that should override the defaults before starting or restarting {product-title}.
```
Comment (eggfoobar) [inline modules/microshift-config-yaml.adoc:54]: same thing here,

> `/etc/microshift/config.yaml` directory,

to

> `/etc/microshift` directory,

**Review by eggfoobar — COMMENTED** [review]
Looks good! Just had some small suggestions

File: modules/microshift-config-yaml.adoc [hunk 3]
```diff
@@ -44,21 +49,22 @@ debugging:
...
+
 [IMPORTANT]
 ====
 Restart {product-title} after changing any configuration settings to have them take effect. {product-title} reads the configuration file only on start.
```
Comment (dhensel-rh) [inline modules/microshift-config-yaml.adoc:58]: A little outside of scope, but could you be more specific of `on start`. Is it on start of MicroShift ? I read that and think start of the system.

**Review by abrennan89 — COMMENTED** [review]
Good work so far! Left some comments

**Comment by abrennan89** [comment]
@ShaunaDiaz don't forget to check the box in the description if this is QE approved 😉 looks like it is from the label

File: microshift_configuring/microshift-using-config-tools.adoc [hunk 4]
```diff
@@ -12,10 +12,10 @@ include::snippets/microshift-greenboot-status-snip.adoc[leveloffset=+2]
...
 
-//[id="additional-resources_microshift-using-config-tools_{context}"]
-//[role="_additional-resources"]
-//== Additional resources
+[id="additional-resources_microshift-using-config-tools_{context}"]
+[role="_additional-resources"]
+== Additional resources
 
-//TODO:add xref to check greenboot status module after merge
+* xref:../microshift-greenboot-checking-status.adoc#microshift-greenboot-checking-status[Checking Greenboot status]
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply microshift_configuring/microshift-using-config-tools.adoc:21]: Huh, would have thought Travis would catch that. Odd.

File: modules/microshift-config-yaml.adoc [hunk 5]
```diff
@@ -6,15 +6,20 @@
...
-//include admonition snippet about waiting for greenboot
+On start up, {product-title} searches the system-wide `/etc/microshift/` directory for a configuration file named `config.yaml`. To use custom configurations, you must create the configuration file and specify any settings that should override the defaults before starting {product-title}.
 
 [id="microshift-yaml-default_{context}"]
 == Default settings
-If you do not create a `config.yaml` file, the default values are used. The following example configuration shows the default settings. You must create a custom `config.yaml` file and change any settings that should override the defaults before starting {product-title}.
+If you do not create a `config.yaml` file, the default values are used. The following example shows the default configuration settings.
 
-.Default YAML values example
+* You can use the following command to see the default values:
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply modules/microshift-config-yaml.adoc:15]: Yeah, I know. It really isn't a procedure for creating custom values, though. MicroShift doesn't make a default yaml, which has been confusing for readers. The command is here to show that these values are given in the output of that command. And also to allow users to copy the output and insert values. If you have a better way to present that, I am open to suggestions!

File: modules/microshift-config-yaml.adoc [hunk 6]
```diff
@@ -44,21 +49,22 @@ debugging:
...
-Restart {product-title} after changing any configuration settings to have them take effect. {product-title} reads the configuration file only on start.
+Restart {product-title} after changing any configuration settings to have them take effect. The `config.yaml` file is read only when {product-title} starts.
 ====
 
-[id="microshift-yaml-custom_{context}"]
-== Custom settings
-You must change any settings that should override the defaults before starting {product-title}.
-
-advertiseAddress::
+[id="microshift-yaml-advertiseAddress_{context}"]
```
Comment (adellape) [inline modules/microshift-config-yaml.adoc:61]: Not blocking merge for this, but `id`s should be all lowercase.

**Review by adellape — COMMENTED** [review]
Out-of-scope for PR, but noticed a table heading elsewhere in the assembly with a punctuation in it: "Ports to avoid."

File: modules/microshift-config-yaml.adoc (hunk 6 above)
Comment (ShaunaDiaz) [reply modules/microshift-config-yaml.adoc:61]: oh yeah, just copied it, thx, will bolo

**Pull request merged**